conf/messages

```diff
@@ -1377,95 +1377,166 @@ reliefDetails.caption=Reliefs
+
 lettingsReliefValue.error.required=Enter an amount of Letting Relief, like 108 or 2,940.54
 lettingsReliefValue.error.tooSmall=Amount of Letting Relief must be an amount of money between 1 and {0}
 lettingsReliefValue.error.tooLarge=Amount of Letting Relief must be an amount of money between 1 and {0}
 lettingsReliefValue.error.invalid=Amount of Letting Relief must be an amount of money, like 108 or 2,940.54
-lettingsRelief.furtherInfoLink=Find out about Letting Relief and how to calculate it (opens in a new window)
 lettingsReliefValue.error.amountOverLimit=Amount of Letting Relief must be an amount of money between 1 and {0}
 lettingsReliefValue.error.amountOverPrivateResidenceRelief=Amount of Letting Relief cannot be higher than the Private Residence Relief
 
 lettingsRelief.cyaChange=if Letting Relief is being claimed
 lettingsReliefValue.cyaChange=the amount of Letting Relief
 
+
 otherReliefs.title=Do you want to claim any other reliefs?
 otherReliefs.agent.title=Does your client want to claim any other reliefs?
 otherReliefs.trust.title=Does the trust want to claim any other reliefs?
+otherReliefs.capacitor.title=Does the person want to claim any other reliefs?
+otherReliefs.personalRep.title=Was the person entitled to claim any other reliefs?
+
 otherReliefs.helpText=You will need to tell us what other reliefs you are claiming and what the total amount of these is.
 otherReliefs.agent.helpText=You will need to tell us what other reliefs your client is claiming and what the total amount of these is.
 otherReliefs.trust.helpText=You will need to tell us what other reliefs the trust is claiming and what the total amount of these is.
-otherReliefsName.label=Names of the other reliefs you are claiming
-otherReliefsName.agent.label=Names of the other reliefs your client is claiming
-otherReliefsName.trust.label=Names of the other reliefs the trust is claiming
-otherReliefsAmount.label=Total amount of other reliefs you are claiming
-otherReliefsAmount.agent.label=Total amount of other reliefs your client is claiming
-otherReliefsAmount.trust.label=Total amount of other reliefs the trust is claiming
+otherReliefs.capacitor.helpText=You will need to tell us what other reliefs the person is claiming and what the total amount of these is.
+otherReliefs.personalRep.helpText=You will need to tell us what other reliefs the person was entitled to claim and what the total amount of these is.
+
 otherReliefs.furtherInfoLink=Find out about reliefs you might qualify for (opens in a new window)
 otherReliefs.agent.furtherInfoLink=Find out about reliefs your client might qualify for (opens in a new window)
 otherReliefs.trust.furtherInfoLink=Find out about reliefs the trust might qualify for (opens in a new window)
+otherReliefs.capacitor.furtherInfoLink=Find out about reliefs the person might qualify for (opens in a new window)
+otherReliefs.personalRep.furtherInfoLink=Find out about reliefs the person might qualify for (opens in a new window)
```

Comment:
this would be more consistent with the other personal rep content if it was framed in the past tense like "might have qualified for" - this is a content questions though